nvme-print.c

```diff
@@ -2947,16 +2982,31 @@ void nvme_show_zns_id_ns(struct nvme_zns_id_ns *ns,
 
 	printf("NVMe ZNS Identify Namespace:\n");
 	printf("zoc:    %u\n", le16_to_cpu(ns->zoc));
+	if (human)
+		show_nvme_id_ns_zoned_zoc(ns->zoc);
 	printf("ozcs:   %u\n", le16_to_cpu(ns->ozcs));
+	if (human)
+		show_nvme_id_ns_zoned_ozcs(ns->ozcs);
 	printf("mar:    %u\n", le16_to_cpu(ns->mar));
 	printf("mor:    %u\n", le16_to_cpu(ns->mor));
 	printf("rrl:    %u\n", ns->rrl);
 	printf("frl:    %u\n", ns->frl);
 
-	for (i = 0; i <= id_ns->nlbaf; i++)
-		printf("lbafe %02d: zsze:%"PRIx64" zdes:%u%s\n", i,
-			(uint64_t)le64_to_cpu(ns->lbafe[i].zsze),
-			ns->lbafe[i].zdes, i == lbaf ? " (in use)" : "");
+	for (i = 0; i <= id_ns->nlbaf; i++){
+		if (human)
+			printf("LBA Format Extension %2d : Zone Size: %"PRIx64" blocks"
+					"Zone Descriptor Extension Size: %-5d %s\n",
```

Comment:
Add bytes (or b, or B) after Zone Descriptor Extension Size. Such that we know it is bytes, and already been multiplied by 64 bytes.